pages/docs/tutorials/kafka/index.md

```diff
@@ -0,0 +1,4 @@
+---
+title: Tutorial-1
```

Comment:
this document should not be here, why is it here?

Reply:
oh, this was just to get the netlify preview link, I'll remove it as soon as it is ready to merge

Reply:
okay, I'll try removing it